tasks/copy.js

```diff
@@ -14,6 +14,7 @@ module.exports = function(grunt) {
   var fs = require('fs');
   var chalk = require('chalk');
   var fileSyncCmp = require('file-sync-cmp');
+  var isWindows = /^win/.test(process.platform);
```

Comment:
Should be: `var isWindows = process.platform === 'win32';` And yes, it's win32 for 64-bit too.